vendor/assets/stylesheets/react-keyed-file-browser.css

```diff
@@ -23,7 +23,8 @@ div.rendered-react-keyed-file-browser div.action-bar {
     flex-grow: 1;
     min-width: 300px;
     margin-left: 10px;
-    white-space: nowrap; }
+    white-space: nowrap;
```

Comment:
Good detective work finding this file, but files under `vendor` shouldn't be changed directly (they're copies of releases from the external libraries we use).

Instead, check out `core.scss` starting at line 431, and add your CSS changes there.